t/sql/mozart.sql

```diff
@@ -0,0 +1,188 @@
+-- Automatically generated, do not edit.
```

Comment:
Is this just a dump from actual database? Where (in code) is it added during testing?